### ISSUE TITLE
rsync creating empty files

### ISSUE THREAD

**Issue by beankylla** [ovh/svfs#55]
Hello,

I am really really enthusiastic about your app! I have been waiting for it for a lonnnng time!

Thanks a lot for all the great work!

I do have one issue.

I tried to copy a big folder by issuing the rsync -r command.

It seems to work well and files get copied (I have the corresponding Network activity on my computer), but when I visit the online hubic page or when I browse the directory on the hubic side, all the files are 0 bytes + have a random 6 letter extension (ex: ".r30nL4 ").

Am I doing anything wrong?

Thanks anyways and have a nice day!

**Comment by xlucas** [comment]
Hello

Thanks :)

Yes some important flags are required to use svfs with rsync since your files are mapped to entire objects and not blocks. The `.XXX` objects you have are blocks rsync is trying to write at certain offsets which is not something natively supported by object stores.  There's a bit of documentation on this in the [FAQ](https://github.com/ovh/svfs/blob/master/docs/FAQ.md#how-does-it-compare-to-hubicfuse-cloudfuse-swiftfs--).

Basically, the key options that allow working with objects are `-W` and `--inplace`. This is also highly recommended to add `extra_attr=true` to mount options and preserve modification times with rsync (`-t` option).

**Comment by beankylla** [comment]
Hello and thanks for the support!

It's just as great as the Software! 

I'm sorry i didn't RTFM :$! Shame on me! :)

Thanks and have a nice day! :)
